entgql/template.go

```diff
@@ -71,6 +71,8 @@ var (
 		"filterNodes":         filterNodes,
 		"findIDType":          findIDType,
 		"nodePaginationNames": nodePaginationNames,
+		"skipMode":            skipModeFromString,
+		"hasSkipMode":         hasSkipMode,
```

Comment:
This function is not needed IMO, you can use the `skipMode` function in the template (0 is falsy).